### DISCUSSION TITLE
CloudKey Testing

### DISCUSSION THREAD

**Discussion by lassealfastsen** [NRCHKB/node-red-contrib-unifi-os#16] [General]
@crxporter i thought i would open a discussion for testing on CloudKey hw.

Let me know if you need me to test anything :)

**Comment by crxporter** [comment]
Yes awesome! Just saw your other issue, glad to know you're up and running. 

Take notes here of what works, what doesn't work, etc. 

We can check in occasionally to discuss what new features we have that you could test. 

What hardware do you have? Any protect cameras?

**Reply by lassealfastsen** [reply]
I am running on a Cloud key G2+ fully updated, and i have protect running with 4 G3 Bullet cams.

For now i am using the following to get the temp from my USG-8

```
    POST /proxy/network/api/s/default/stat/device
    {
        "macs": [
            "b4:fb:e4:57:57:7e"
        ]
    }
```

this works without a hitch so far.

Otherwise my setup is fairly small.

1 x USG3P
1 x USW-8 (150W)
1 x USW-16 PoE
3 x UAP-AC-Pro
1 x Cloud Key G2+
